### ISSUE TITLE
Checksums in release notes

### ISSUE THREAD

**Issue by jmfgdev** [MartinBriza/MediaWriter#143]
It would be good If gpg signed checksums were provided somehow, maybe in the release description, or along with the release.

**Comment by MartinBriza** [comment]
That's true. I'll fix this problem for 4.1.1 and will continue releasing the sums in the future.

**Comment by asd-collab** [comment]
can't find the checksums. where are the release notes?

are they like this-> https://github.com/VSCodium/vscodium/releases/download/1.40.1/VSCodium-win32-x64-1.40.1.zip.sha256, cause its going to be another unnecessary process to separate the SHA256 hashes from unneeded stuff.

How about doing it like Ubuntu ISOs. Checksums and GPG signs? that sounds good and flexible enough for most use-cases. And don't forget the public key for GPG signs.

---

 asd-collab
